src/main/java/com/sap/cloudfoundry/client/facade/CloudControllerClient.java

```diff
@@ -88,7 +88,19 @@ void bindServiceInstance(String applicationName, String serviceInstanceName, Map
     void createApplication(String applicationName, Staging staging, Integer disk, Integer memory, Set<CloudRouteSummary> routes);
 
     /**
-     * Create a service instance.
+     * Create application.
+     *
+     * @param applicationName application name
+     * @param staging staging info /** Create application.
+     *
+     * @param applicationName application name
+     * @param staging staging info
+     * @param disk disk quota to use in MB
+     * @param memory memory to use in MB
+     * @ping staging, Integer disk, Integer memory, Set<CloudRouteSummary> routes, DockerInfo dockerInfo); void createApplication(String
+     *       applicationName, Staging staging, Integer disk, Integer memory, Set<CloudRouteSummary> routes);
```

Comment:
Isn't this method commented out?

Reply:
nice catch, this mistake was done during merge conflict resolving